App.tsx

```diff
@@ -48,9 +55,9 @@ const App = () => {
     }
   }
 
-  async function startFitnessAssessment(){
+  async function startFitnessAssessment(type:AssessmentTypes, showSummary:boolean, customAssessmentID:string){
```

Comment:
@SencyNetanel please split into multiple lines and add an inline comment that explains each input argument